src/meta/user/userRoles.ts

```diff
@@ -1,5 +1,16 @@
+import { RoleName } from './userRole'
+
 const noRole = { role: 'NONE', labelKey: 'user.roles.noRole' }
 
+const roleNames = [
```

Comment:
This is not correct , because it doesn't include all roles .
This is not a global object .
These are these are the role names used in admin view. Create an object there if you need

Reply:
done, the array is defined into UserList and it is passed via component props